syntax = "proto3";

package foundationdb;

service NetworkTest {
	rpc GetValue(NetworkTestRequest) returns (NetworkTestReply) {}
}

message NetworkTestRequest {
	bytes key = 1;
	uint32 replySize = 2;
}

message NetworkTestReply {
	bytes value = 1;
}
